drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/hmis_csv_cleanup/delete_empty_enrollments.rb

```diff
@@ -0,0 +1,48 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module HmisCsvTwentyTwenty::HmisCsvCleanup
+  class DeleteEmptyEnrollments < Base
+    def cleanup!
+      enrollment_batch = []
+
+      enrollments_with_no_service = enrollment_scope.where.not(id: enrollment_scope.joins(:services).select(:id))
+      enrollments_with_no_cls = enrollment_scope.where.not(id: enrollment_scope.joins(:current_living_situations).select(:id))
+      empty_enrollments = enrollment_scope.
+        where(id: enrollments_with_no_service.select(:id)).
+        where(id: enrollments_with_no_cls.select(:id))
```

Comment:
Potentially this fixes the above since it would need to be in both sets, but I bet this query will be relatively slow.